tests/unit/specs/components/App.spec.js

```diff
@@ -2,7 +2,7 @@ import Vue from 'vue';
 import { shallowMount, createLocalVue } from '@vue/test-utils';
 import Vuex from 'vuex';
 import VueI18n from 'vue-i18n';
-import Vocabulary from '@creativecommons/vue-vocabulary/vue-vocabulary.common';
+import Vocabulary from '@creativecommons/vocabulary-components';
```

Comment:
For consistency, should we call it `VueVocabulary` here as well?

Reply:
done